src/elements/emby-checkbox/emby-checkbox.js

```diff
@@ -6,6 +7,11 @@ import 'webcomponents.js/webcomponents-lite';
 const EmbyCheckboxPrototype = Object.create(HTMLInputElement.prototype);
 
 function onKeyDown(e) {
+    // Ignore GamepadA onKeyDown if using gamepadtokey since it will click the checkbox onKeyUp.
+    if (appSettings.enableGamepad && e.key === 'GamepadA') {
+        return;
+    }
```

Comment:
Doesn't it click anyway by this code?
https://github.com/jellyfin/jellyfin-web/blob/5046b1df0ff862de822ded654a405a913c76f940/src/scripts/keyboardNavigation.js#L212-L214